e2e/rbd.go

```diff
@@ -2026,6 +2028,41 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					e2elog.Failf("failed to get server version: %v", err)
 				}
+
+				deploymentPatch, err := json.Marshal(map[string]interface{}{
```

Comment:
Why use a patch, and not just `Get()` and `Update()`. There should not be a need to list all `args` that way.

Reply:
Because we need this for both helm and non-helm deployments.

Reply:
You can just use the Kubernetes API to get the Deployment object, modify the `args` and send the updated object to the API server. No need for JSON and untyped maps of strings.

Reply:
Sure, I can give it a shot, thanks for the hint